inst/tools/ch_minc_depends.m4

```diff
@@ -81,7 +81,7 @@ AC_DEFUN([INSTALL_LIBMINC], [
 	
 	AS_IF([test ! -d libminc], [
 	   echo Downloading libminc
-	   git clone --recursive https://github.com/BIC-MNI/libminc
+	   git clone --branch develop --recursive https://github.com/BIC-MNI/libminc
```

Comment:
This should probably reference the hash, the `configure` file should be autogenerated.

Reply:
Ah, right that's how this works.

That means configure shouldn't be stored inside git though... it should be generated when needed.

Reply:
Sorry I forgot that I changed those lines too.